zlibWrapper/Makefile

```diff
@@ -7,8 +7,9 @@
 
 # Paths to static and dynamic zlib and zstd libraries
 ifneq (,$(filter Windows%,$(OS)))
-STATICLIB = ../../zlib/libz.a ../lib/libzstd.a
-IMPLIB    = ../../zlib/libz.dll.a ../lib/libzstd.a
+ZLIBDIR = ../../zlib1.2.8
```

Comment:
I was asking about whole zstd package. Do we support CygWin compilation? I didn't found any code that detects CygWin so it may throw compilation errors.